.jcheck/conf

```diff
@@ -4,7 +4,8 @@ jbs=JDK
 version=26
 
 [checks]
-error=author,committer,reviewers,merge,issues,executable,symlink,message,hg-tag,whitespace,problemlists,copyright
+#error=author,committer,reviewers,merge,issues,executable,symlink,message,hg-tag,whitespace,problemlists,copyright
+error=author,committer,executable,symlink,message,hg-tag,whitespace,problemlists,copyright
 warning=issuestitle,binary
 
 [repository]
```

Comment:
I think you can update the `branches=.*` field below, maybe make it `branches=lworld` for an exact match only?

Reply:
No, we allow arbitrary branch names in the Valhalla repository, this pr target is for `lw5` . Not sure I follow.